### PR TITLE
raw3-6 bringup + ur10

### PR DESCRIPTION
**Pull request by benmaidel**
updated raw3-6 bringup + added ur10 to bringup

### CODE REVIEW

**Comment by floweisshardt** [comment]
Travis reports an error in the URDF. please fix

**Comment by floweisshardt** [comment]
just merged https://github.com/ipa320/cob_calibration_data/pull/104 and retriggered travis

File: cob_bringup/robots/raw3-6.xml [hunk 1]
```diff
@@ -44,23 +45,29 @@
...
 
-                <include file="$(find cob_bringup)/tools/battery_monitor.launch" >
-                        <arg name="robot" value="$(arg robot)" />
-                </include>
+		<include file="$(find cob_bringup)/tools/battery_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
 
 		<!-- upper robot -->
+		<include file="$(find ur_bringup)/launch/ur_common.launch">
```
Comment (NadiaHG) [inline cob_bringup/robots/raw3-6.xml]: This launch file includes also a robot_state_publisher node https://github.com/ros-industrial/universal_robot/blob/indigo-devel/ur_bringup/launch/ur_common.launch#L28 

`roslaunch file contains multiple nodes named [/robot_state_publisher]`

**Review by NadiaHG — CHANGES_REQUESTED** [review]

File: cob_bringup/robots/raw3-6.xml (hunk 1 above)
Comment (benmaidel) [reply cob_bringup/robots/raw3-6.xml]: thx already fixed

**Comment by floweisshardt** [comment]
strange, but travis reports an issue about raw3-4:

```
[/home/travis/catkin_ws/src/cob_robots/cob_bringup/robots/raw3-4.xml]:
    included file [/home/travis/catkin_ws/src/cob_robots/cob_bringup/drivers/ur.launch] requires the 'min_payload' arg to be set
```

**Comment by benmaidel** [comment]
@ipa-nhg @ipa-fmw can someone please review / merge

**Review by NadiaHG — APPROVED** [review]
FR: OK

**Pull request merged**